.golangci.yaml

```diff
@@ -32,6 +32,17 @@ linters-settings:
       - prefix(github.com/aquasecurity/)
       - blank
       - dot
+  gomodguard:
+    blocked:
+      modules:
+        - github.com/hashicorp/go-version:
+            recommendations:
+              - github.com/aquasecurity/go-version
+            reason: "`aquasecurity/go-version` is degisned for our use-cases"
```

Comment:
```suggestion
            reason: "`aquasecurity/go-version` is designed for our use-cases"
```

Reply:
Thanks! fixed in https://github.com/aquasecurity/trivy/pull/5228/commits/556e3e8b849297c5eea16526bedbfa09b2dafea6